web/custom-plugins/forecast-of-response/src/components/Operations/PhaseView.tsx

```diff
@@ -522,19 +587,22 @@ function SingleSimulationConfig(props: {
             </option>
           ))}
         </StyledSelect>
-        {results[selectedResult]?._id && (
-          <Result
-            result={results[selectedResult]}
-            addPlotWindow={addPlotWindow}
-          />
-        )}
-        {resultGraphs &&
-          Object.keys(resultGraphs).map((index: string) => (
+        {plotWindows &&
```

Comment:
```suggestion
        {
```
I dont see how this can every be false.
Empty object is "true".